src/components/Password/Password.tsx

```diff
@@ -0,0 +1,76 @@
+import React, { useState, ComponentPropsWithoutRef } from "react";
+import { FiEyeOff, FiEye } from "react-icons/fi";
+
+export interface PasswordProps extends ComponentPropsWithoutRef<"button"> {
+  placeholder?: string;
+}
+
+export function Password(props: PasswordProps) {
+  const { placeholder } = props;
+
+  let iconClass = `w-5 h-5 inline-block align-text-bottom text-neutral`;
```

Comment:
Should setting tailwind classes in variable be a standard for our implementation ?

Reply:
This is not mandatory but it would be easy to understand IMHO! Have a look how I split them by concerns: focus, hover, etc...